go/vt/vtgate/engine/mstream.go

```diff
@@ -62,7 +61,7 @@ func (m *MStream) GetTableName() string {
 
 // TryExecute implements the Primitive interface
 func (m *MStream) TryExecute(ctx context.Context, vcursor VCursor, bindVars map[string]*querypb.BindVariable, wantfields bool) (*sqltypes.Result, error) {
-	return nil, vterrors.New(vtrpcpb.Code_INTERNAL, "[BUG] 'Execute' called for Stream")
+	return nil, vterrors.VT13001("TryExecute is not supported by MStream")
```

Comment:
Similar here, should we call describe this as `TryExecute is not supported for a message stream` instead of using the more internal `MStream` name?

Reply:
Sounds good to me, resolved in fa32cade5f